project/Dependencies.scala

```diff
@@ -27,6 +27,7 @@ object Dependencies {
     // If changing this, remember to update protoc plugin version to align in
     // maven-plugin/src/main/maven/plugin.xml and akka.grpc.sbt.AkkaGrpcPlugin
     val googleProtobuf = "3.21.9" // checked synced by VersionSyncCheckPlugin
+    val googleApi = "2.15.0" // checked synced by VersionSyncCheckPlugin
```

Comment:
```suggestion
    val googleApi = "2.15.0" 
```
Or actually add it to the version sync check plugin for checking but I'm not sure that is needed.